boards/arm/frdm_kw41z/frdm_kw41z.dts

```diff
@@ -61,6 +61,33 @@
 			gpios = <&gpioc 5 GPIO_INT_ACTIVE_LOW>;
 		};
 	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <1 0 &gpiob 18 0>,	/* A1 */
```

Comment:
Should we add a comment on why A0 is missing?

Reply:
That would be consistent with existing practice at https://github.com/zephyrproject-rtos/zephyr/blob/132130d17a93b85479162ea2a14ded28f7ca7019/boards/riscv/hifive1_revb/hifive1_revb.dts#L46

Reply:
Comment added